docker-compose.yml

```diff
@@ -38,13 +38,17 @@ services:
       SCHEMA_REGISTRY_LISTENERS: http://schema-registry:8081
 
   tools:
+    depends_on:
+      - "kafka"
+      - "zookeeper"
+      - "schema-registry"
     image: cnfltraining/training-tools:19.06
     hostname: tools
```

Comment:
Another issue I confronted was that hostnames set in the compose file did not work on host (at least not out of the box).
Afaik, docker hostname can only be resolved inside Docker, they are not visible to host. Because sbt is run on host,
all those http address using these host names could not be resolved.

I'm running Ubuntu 20.04 so I just worked this around by aliasing `kafka` and `schema-registry` hosts in `/etc/hosts` file. After those hosts could be resolved to docker services, running those sbt projects worked fine.

Reply:
So `etc/hosts` is the way to go for exposing docker hostname to host machine. I don't recall setting it by hand however I have entires for the defined services there.
Maybe it's Docker Desktop for Mac which does that automatically? I am not sure. I will add a info about `etc/hosts` to the `README` as well.